versions-rules.xml

```diff
@@ -0,0 +1,4 @@
+<ruleset comparisonMethod="maven"
+         xmlns="http://mojo.codehaus.org/versions-maven-plugin/rule/2.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://mojo.codehaus.org/versions-maven-plugin/rule/2.0.0 http://mojo.codehaus.org/versions-maven-plugin/xsd/rule-2.0.0.xsd">
+</ruleset>
```

Comment:
Is this file needed?

Reply:
Yes, for the Maven versions plugin, otherwise `versions:update-parent` complains that there is no version rules.